
string[] name
float64[] position
float64[] velocity
float64[] effort
float64[] stiffness
float64[] damping

string aux_name
float64[] aux

int32 seq_id